### DISCUSSION TITLE
How to train multiple subjects using this specific model?

### DISCUSSION THREAD

**Discussion by dvelm** [TheLastBen/fast-stable-diffusion#822] [Q&A]
Hello, this new dreambooth version is very nice and can give very good outputs. But the problems appear when I want train multiple subects using this model https://huggingface.co/prompthero/openjourney/resolve/main/mdjrny-v4.ckpt
If I train only one person it gives very nice outputs, no glitches, no errors, no distorsion. But if I choose more than one, the pictures come out ruined with black spots.
I tried training one subject and then with the new created model re-training a new subject, but only the new one comes with good result, the fist one comes with black spots. I tried also to train all the subjects in the same time using different name for the pictures (..01, ..02,..03,...) but in this way all the outputs are not good.
Should I train the model one time for every subjects? In this way it works. Or maybe I should use a different percentage for "Train_text_encoder_for:", I usually use 50. Or the problem it's the option "Contains_faces" and it's better to leave no as preference?
Any suggestion, with this specific model. It gives surprising results when the images are correctly trained.

**Comment by TheLastBen** [comment]
Hi,
Don't use the contains_faces option and reduce the text_encoder %

**Reply by dvelm** [reply]
Thanks. I will try it

**Reply by dvelm** [reply]
Just one other question. Is it reccomended to train all the images in one run or it can be done with one subject at time using the new created .ckpt every time?

**Reply by TheLastBen** [reply]
it is preferable to train multiple subjects (different genders) at the same time as they act as some sort of self-regularization.

**Reply by dvelm** [reply]
after many tries I noticed that the model isn't loading if I deactivate face option. After 3 hours it still says loading weights.
I have chosen Contains_faces no, text_encoder 40, fp16 yes and no, but same result, the model is not loading in automatic1111.
If I activate Contains face the model gets loaded. 
What if I uncheck text encoder, does it work?

**Reply by dvelm** [reply]
just an update, the model is loaded fine after setting the text encoder and disabling containing face
